packages/strapi-design-system/src/v2/LinkButton/LinkButton.stories.mdx

```diff
@@ -0,0 +1,153 @@
+<!--- LinkButton.stories.mdx --->
+
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+import { ArgsTable } from '@storybook/addon-docs';
+import Information from '@strapi/icons/Information';
+import Write from '@strapi/icons/Write';
+import { LinkButton } from './LinkButton';
+import { Box } from '../../Box';
+import { Flex } from '../../Flex';
+
+<Meta title="Design System/Components/v2/LinkButton" component={LinkButton} />
+
+# LinkButton
+
+LinkButtons are a way to perform actions that links to a different page component that will stay within one page. They help interact with its content via a single click. According to their style, specific labels and icons, they help understand the most important actions to take.
+
+The difference with the `Button` component will be that the action of the LinkButton redirects to a different page.
+The difference with the `Link` is the style.
+
+Best practices
+
+- A link button should have a relevant and clear label.
+- Be positioned in consistent locations in the interface.
+- If there are no primary action on a screen yet, this might be the one to look for.
+- Prefer to have not more than two buttons side to side.
+- To navigate to another page in your browser (use Link component instead).
+
+[View source](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/v2/LinkButton)
+
+## Imports
+
+```js
+import { LinkButton } from '@strapi/design-system/v2/LinkButton';
+```
+
+## Usage
+
+The most common LinkButton component used in the interface (primary action). It can have different variants and meaning.
+
+Use the `to` parameter to redirect using the router's routes, `href` parameter for an external url.
+
+<Canvas>
+  <Story name="base">
+    <Box paddingBottom={2}>
+      <LinkButton variant="default" href="https://strapi.io/">
```

Comment:
I think we are missing a target logic for this, we have target self even with href 🤔

<img width="362" alt="image" src="https://user-images.githubusercontent.com/71838159/158580883-49b65feb-70eb-4751-86d1-9827733597f3.png">

Reply:
`_self` is the default target attribute for anchor tags. For the v2 components, you need to pass `isExternal` to define that the link is an external link (`isExternal` will set the target to `_blank`. We can't use `href` to define that the link is external because with NextJS, href is used for internal links too.)

Reply:
I see makes sense, thank you for adding it in the Story, will be easier to remember!